esm/inverse_folding/util.py

```diff
@@ -112,10 +112,10 @@ def get_sequence_loss(model, alphabet, coords, seq):
     coords, confidence, strs, tokens, padding_mask = batch_converter(
         batch, device=device)
 
-    prev_output_tokens = tokens[:, :-1]
+    prev_output_tokens = tokens[:, :-1].to(device)
```

Comment:
@naailkhan28 what is the rationale for this change? Passing `device` into `model.forward()` seems like a more general fix than moving the tensor to the device beforehand.

Reply:
Actually, removing `device` from `model.forward()` is needed. My evaluation setup was improperly configured, so I was not catching the error.